CMakeLists.txt

```diff
@@ -198,26 +175,50 @@ set_target_properties(nova-renderer PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
 target_compile_definitions(nova-renderer PUBLIC DUMP_NOVA_CALLS NOMINMAX)
 #endif()
 
-set(COMMON_LINK_LIBS ${CMAKE_DL_LIBS} spirv-cross-core spirv-cross-glsl spirv-cross-hlsl spirv-cross-reflect spirv-cross-util nova-profiler miniz glslang SPIRV ${SPIRV_SHARED_LIBRARIES} ${Vulkan_LIBRARIES} ${GLFW_LIBRARY_NAME})
-if (WIN32)
+set(COMMON_LINK_LIBS ${CMAKE_DL_LIBS}
+                     glm::glm
+                     nlohmann::json
+                     spirv::headers
+	                 spirv-cross-core
+	                 spirv-cross-glsl
+	                 spirv-cross-hlsl
+	                 spirv-cross-reflect
+	                 spirv-cross-util
+	                 nova-profiler
+	                 miniz
+	                 glslang
+	                 SPIRV
+	                 ${SPIRV_SHARED_LIBRARIES}
+	                 ${Vulkan_LIBRARIES}
+	                 ${GLFW_LIBRARY_NAME})
```

Comment:
We don't use GLFW. I guess it shuck into the original CMake file but it should be removed, especially since you removed the GLFW submodule